src/chainparams.cpp

```diff
@@ -419,7 +419,7 @@ class CRegTestParams : public CChainParams {
         pchMessageStart[1] = 0xbf;
         pchMessageStart[2] = 0xb5;
         pchMessageStart[3] = 0xda;
-        nDefaultPort = 18444;
+        nDefaultPort = 33556;
```

Comment:
Wouldn't it be annoying to have that different in 1.14 if you wanted to test interactions between them?

Reply:
Possibly. Mostly I don't care. Will revert :D